app/models/organization.rb

```diff
@@ -59,7 +59,6 @@ class Organization < ApplicationRecord
   has_one :salesforce_integration, class_name: "Integrations::SalesforceIntegration"
 
   has_one :applied_dunning_campaign, -> { where(applied_to_organization: true) }, class_name: "DunningCampaign"
-  has_one :default_billing_entity, -> { active.order(created_at: :asc) }, class_name: "BillingEntity"
```

Comment:
I needed to change this one from association to method, because the association remains cached, while method is always reloaded. Now in tests we create a billing entity while creation organization, and since we have an additional logic for default_currency and timezone, it slows creation of organization for micro-seconds, but that's enough for rails to create them with less delay, and in order to get org's default_billing_entity in tests, the organization should be reloaded. Instead, having it always reloaded will be more safe

Reply:
at the same time, technically we will remove this additional logic... so maybe we can keep the scop, but add reload in after create for organization factory...